ns3777k/prometheus-bundle/1.0/config/packages/ns3777k_prometheus.yaml

```diff
@@ -0,0 +1,7 @@
+ns3777k_prometheus:
+    namespace: app
+    adapter: apcu
+
+    listener:
+        enabled: true
```

Comment:
This line is not needed, right? Or do you expect a large number of the users to disable this?

Reply:
not sure. i removed the line :smile: